packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -213,7 +212,7 @@ async function downloadNordigenTransactions(
 
     return {
       transactions: booked,
-      accountBalances: balances,
+      accountBalance: balances,
```

Comment:
This is a runtime changes as I've found a mismatch: consumers expect `accountBalance` but we return `accountBalances`. I picked one 🤷

Reply:
fyi @fstybel — you touched this last, wanted to make sure this change is as expected.